adapters/zio-http/src/test/scala/caliban/ZHttpAdapterSpec.scala

```diff
@@ -53,7 +54,14 @@ object ZHttpAdapterSpec extends ZIOSpecDefault {
     suite.provideShared(
       apiLayer,
       Scope.default,
-      Server.defaultWith(_.port(8089).responseCompression())
+      Server.defaultWith(
+        _.port(8089)
+          .withWebSocketConfig(
```

Comment:
Well that sucks. It doesn't work without it, right?

Reply:
Nope 😞 I wonder whether we can create a default websocket config in the `adapters/zio-http` module and advice users that they need to use it.

By the way I think this might be a good thing for users to be able to customise. In the past I wanted to disable legacy WS protocol support in a service, and I had to write a custom middleware to do so. Being able to do that via the server config can be handy. Although it's a bit annoying that we can't provide a default without users making changes to their code

Reply:
Yeah it would have been nice if it worked by default and then we could override it by config.

Reply:
Yeah that would have been nice indeed. As a middleground, I added a default websocket config to the ZHttpAdapter which I hope it'll improve UX a bit

Reply:
What's your opinion on this btw https://discord.com/channels/629491597070827530/633200096393166868/1113331097120555029